### ISSUE TITLE
Beta 3.9.0-rc 

### ISSUE THREAD

**Issue by Grateful-Viv** [code4recovery/12-step-meeting-list#201]
Updated to WP v5.5.2. and then installed TSML v3.9.0-rc 

Though the meetings were visible after activating this version (yes, did de-activate previous version), trying to view the meeting returned me to home page. Trying to view meetings through the admin panel yielded no meetings visible. Had to reinstall all meetings from CSV file.

Other than very visible jQuery Migrate Helper — Warnings encountered issue stating _/wp-content/plugins/12-step-meeting-list-3.9.0-rc.1/assets/js/admin.min.js: jQuery.parseJSON requires a valid JSON string_ everything else seems to be fine.

It's a nice feature to not have to put an actual street address for online meetings (these meetings have never met indoors as opposed to the ones that are relocated online), and with the update of the Meeting Guide app in which it will support online-only meetings with an approximate location (e.g. county/state), this will make more visual sense to people. See https://meetingguide.aa.org/online-meetings.

**Describe the bug**

**To Reproduce**

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**

**Comment by tech2serve** [comment]
*** Warning: For anyone else reading this thread, these steps are not intended for non-beta testing users. Please do not make changes to your database except through the plugin ***

Thanks @Grateful-Viv. 

Here are the steps I took in an attempt to reproduce the issue you experienced:

1. I removed all meetings from my database and rolled TSML back to v3.8.0. I also removed references in the `wp_options` table that would have been updated through the beta testing process. I think reloaded my data from my csv. *In doing these steps, I returned my system to the same state as non-beta testers*

1. I then updated to Wordpress 5.5.2. 

1. Next, I (deactivated v3.8.0 of the plugin and) upgraded to "v3.9.0-rc.1".

I did experience any issues with the meetings, and I confirmed the correct changes occurred in the database.

*What does this mean?*

I'm hopeful that what you experienced was a fluke resulting from the multiple tests you've run. If you wish to confirm this, you can follow the steps I outlined above on your dev/staging site.

**Comment by Grateful-Viv** [comment]
1. I rolled back to a backup copy with v3.8.0. and verified everything good.
2. Uploaded all meetings from a CSV file and verified everything good.
3. Updated WordPress to v 5.5.3 (yes, they changed it again!) and verified everything good.
4. Then updated plugins (not TSML) and verified everything good.
5. Then deactivated v3.8.0, uploaded and activated v3.9.0-rc.1
6. All looked good, even after doing a bunch of edits. 

Thank you.
